manifests/m/Microsoft/WindowsAppSDKRuntime/1.0/Microsoft.WindowsAppSDKRuntime.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsAppSDKRuntime
+PackageVersion: 1.0
+InstallerSwitches:
+  Silent: -q
+  SilentWithProgress: -q
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://aka.ms/windowsappsdk/1.0/latest/windowsappruntimeinstall-1.0-x86.exe
+  InstallerSha256: D339291975B86821F06351DCA7A8F3802101BB9E3A2F822271338AD7258BDFEB
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://aka.ms/windowsappsdk/1.0/latest/windowsappruntimeinstall-1.0-x64.exe
+  InstallerSha256: 81170D87B5218CB2BF3002FE4918FCDBF99B021BA96A5B9E1E20263D6E58383E
+- Architecture: arm64
+  InstallerType: exe
+  InstallerUrl: https://aka.ms/windowsappsdk/1.0/latest/windowsappruntimeinstall-1.0-arm64.exe
+  InstallerSha256: 6D261F24A7EEB255B71EA506E595486759AA9DCB345EE9CB568C13BA2F8429E0
+ManifestType: installer
+ManifestVersion: 1.1.0
```

Comment:
Should this be 1.0.1 ?

Reply:
> Should this be 1.0.1 ?

Notice the link https://aka.ms/windowsappsdk/1.0/latest/... That'll grab the latest patch for 1.0. Today that's 1.0.1, in the future it can be 1.0.2, 1.0.3, 1.0.789... ;-)

Reply:
@kevinlams is ManifestVersion for the WinAppSDKRuntime version, or a WinGet version number?

Reply:
Never mind. Looks like ManifestVersion is nothing to do with the package's version.